sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -582,6 +582,15 @@ object SQLConf {
     .intConf
     .createWithDefault(2667)
 
+  val CODEGEN_MAX_CHARS_PER_FUNCTION = buildConf("spark.sql.codegen.maxCharactersPerFunction")
```

Comment:
In this PR, do we change this parameter to use `number of lines` instead of `number of characters`, too?